cubids/cli.py

```diff
@@ -177,6 +178,17 @@ def _parse_validate():
         help="Lets user run a locally installed BIDS validator. Default is set to False ",
         required=False,
     )
+    parser.add_argument(
```

Comment:
This is a great idea. Note to future self: add section to docs that show where to get these files and suggest that users grab a copy at the start of a project.